ts/webui/src/components/experiment/trialdetail/table/tableFunction/search/searchFunction.ts

```diff
@@ -1,7 +1,7 @@
 import { mergeStyleSets } from '@fluentui/react';
-import { trialJobStatus } from '../../../static/const';
-import { EXPERIMENT } from '../../../static/datamodel';
-import { TableObj, SearchItems } from '../../../static/interface';
+import { trialJobStatus } from '../../../../../../static/const';
```

Comment:
Could you try absolute-path import? The import path looks really long and strange.

Reply:
https://github.com/microsoft/nni/blob/master/ts/webui/scripts/config/webpack.config.dev.js#L30
https://github.com/microsoft/nni/blob/master/ts/webui/scripts/config/webpack.config.js#L255

Change above two lines to `modules: [ 'node_modules', 'src' ],`

Reply:
I try to resolve this issue according [the reference](https://juejin.cn/post/6892636779850137608) but not succeed. It's so easy to resolve by change two lines! Thanks for your help. :)